### PR TITLE
Release 3.3.0-BETA

### PR DESCRIPTION
**Pull request by heyams**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+# Version 3.3.0-BETA
+* Update OpenTelemetry to 1.14 [#2257](https://github.com/microsoft/ApplicationInsights-Java/pull/2257).
```
Comment (trask) [inline CHANGELOG.md:4]: we have only updated to 1.14-SNAPSHOT so far (the fork versioning is confusing sorry)
```suggestion
* Update OpenTelemetry to 1.13 [#2257](https://github.com/microsoft/ApplicationInsights-Java/pull/2257).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+* Deprecate logs' LoggingLevel custom property [#2254](https://github.com/microsoft/ApplicationInsights-Java/pull/2254).
+* Add support for OpenTelemetry log exporter and metric exporter.
```
Comment (trask) [inline CHANGELOG.md:6]: users won't see a difference related to logs (there is no logs API, only auto-instrumentation + logs SDK)
```suggestion
* Add support for OpenTelemetry metrics API
```

**Review by trask — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (heyams) [reply CHANGELOG.md:4]: https://github.com/microsoft/ApplicationInsights-Java/pull/2257/files#:~:text=val%20otelInstrumentationAlphaVersion%20%3D%20%221.14.0.1%2Dalpha%22 

it's 1.14 though?

Comment (trask) [reply CHANGELOG.md:4]: 1.14 instrumentation hasn't been released yet

Comment (heyams) [reply CHANGELOG.md:4]: 1.14.0.1-alpha, i'm confused. does it still get counted as 1.13 since it's alpha? should we say 1.14.0.1-alpha instead?

**Pull request merged**